tasks/collectors/run-collectors/run-collectors.yaml

```diff
@@ -60,6 +64,8 @@ spec:
                 arg_value="$(jq -r '.value' <<< "$param")"
                 ARGS=("${ARGS[@]}" --"${arg_name}" "${arg_value}")
             done
+            # add the previous release json path as an arg
+            ARGS=("${ARGS[@]}" --"previousRelease" "$(params.previousReleasePath)")
```

Comment:
Nit: No need for `""` in `--"previousRelease"` - here it's a literate string, no substitution. Just saying, ok to keep.

Reply:
But also, what about the scenario where it's empty? Is that the responsibility of the collector python script to handle that?

And also, you should add the `$(workspaces.data.path)` prefix, no?

Reply:
> Nit: No need for `""` in `--"previousRelease"` - here it's a literate string, no substitution. Just saying, ok to keep.

yep. thanks. bad copy/paste

Reply:
> And also, you should add the `$(workspaces.data.path)` prefix, no?

right again! thanks. amended.

Reply:
Thanks.